src/istgt_lu_disk.c

```diff
@@ -269,9 +270,24 @@ find_first_bit(ISTGT_LU_CMD_Ptr lu_cmd)
 				:"=a" (res), "=&c" (d0), "=&D" (d1)
 				:"0" (0ULL), "1" (nbits), "2" (addr),
 				[addr] "r" (addr) : "memory");
-		res_final +=res;
-		if((unsigned long)res != nbits)
+		res_final += res;
+		if ((unsigned long)res != nbits)
 			return res_final;
+#elif defined(__aarch64__)
```

Comment:
Also there is no __aarch64__ specific code here. Lets put this in #else section so that it works for all other architecture.

Reply:
Oh yes , I am working on these codes.